### DISCUSSION TITLE
change file format

### DISCUSSION THREAD

**Discussion by kosakepferd** [gsantner/markor#2066] [Q&A] [answered]
Can someone explain to me what it does when I change the format of txt files to markdown?
And then do I have to change the file extension to md as well?

Thanks for helping!

**Comment by gsantner** [comment, marked as answer]
It changes the way Markor presents it to you. The syntax highlighter, buttons and view-mode then uses i.e. the plaintext format - so only generic things. You don't have to change the filename/extension for changing/reverting the filetype selection.

Nothing bad that can happen, you can always go back to Markdown for that file again.

**Reply by kosakepferd** [reply]
Okay. And is it possible to convert multiple files (in my case over 500 files) from txt to markdown?

**Reply by gsantner** [reply]
if you mean by "convert" just to make it load in Markdown format: By mass renaming files to .md file extension should be the easiest option in this case. There are sure some tools/applications that can do this in no time.

**Reply by kosakepferd** [reply]
OK, thanks :)
